medmodels/medrecord/_overview.py

```diff
@@ -139,43 +164,59 @@ def _extract_string_attribute_info(
         return values_string
 
 
-def prettify_table(table_info: pl.DataFrame) -> List[str]:
-    """Takes a DataFrame and turns it into a list for printing a pretty table.
+def prettify_table(
+    data: Dict[Union[Group, Literal["Ungrouped"]], AttributeInfo], header: List[str]
```

Comment:
as far as i can see the `Literal["Ungrouped"]` case is created nowhere and is also handled nowhere. It is also ambivalent in the case a group is called "Ungrouped". But because the difference isn't handled anyway, there is no type error